source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -486,9 +542,35 @@ namespace WorldBuilder
                     }
                   else
                     {
-                      // use half-space cooling model for the bottom (side 1) of the slab
-                      temperature = background_temperature + (min_temperature - background_temperature) *
-                                    std::erfc(adjusted_distance / (2 * std::sqrt(thermal_diffusivity * effective_plate_age)));
+                      // use half-space cooling or plate model for the bottom (side 1) of the slab
+                      if (reference_model_name=="plate model")
+                        {
+                          if (adjusted_distance < max_depth)
+                            {
+                              const double plate_velocity_UI = plate_velocity / seconds_in_year;
+                              temperature = background_temperature + (min_temperature - background_temperature) * (1 - adjusted_distance / max_depth);
+                              // todo: the second operator is either "+" or "-" here
```

Comment:
Todo?

Reply:
I think the sign that is there is correct  otherwise the plots would no look correct - but maybe Haoyuan knows better what her was referring to.